### ISSUE TITLE
4.7.3: Spell Checker blocks editing when remote spell checker is offline

### ISSUE THREAD

**Issue by berenddeboer** [WebSpellChecker/ckeditor-plugin-wsc#46]
When you go to "Check Spelling" and www.webspellchecker.net is unavailable, you get the dialog box saying "Spell Checker" in the title and the "Cancel" button at the bottom, and otherwise a blank page.

But the real problem is that you cannot close this dialog box.

There is a JavaScript error in the console:

```
  Uncaught TypeError: Cannot read property 'setDisable' of null
    at Object.<anonymous> (wsc.js?t=H8DA:91)
    at CKEDITOR.dialog.n (ckeditor.js?oyees7:10)
    at CKEDITOR.dialog.CKEDITOR.event.CKEDITOR.event.fire (ckeditor.js?oyees7:12)
    at CKEDITOR.dialog.<anonymous> (ckeditor.js?oyees7:587)
    at CKEDITOR.dom.element.n (ckeditor.js?oyees7:10)
    at CKEDITOR.dom.element.CKEDITOR.event.CKEDITOR.event.fire (ckeditor.js?oyees7:12)
    at HTMLAnchorElement.<anonymous> (ckeditor.js?oyees7:64)
````

The fix would be to catch this error.

Apologies for not submitting the I suppose patch, as the uncompressed version doesn't work with my Drupal setup (Drupal 7 + wysiwyg module), and the compressed js is unreadable.

The problem is easy to verify: just add a line like `127.0.0.1       www.webspellchecker.net` to your `/etc/hosts` file and you can demonstrate the problem.

PS: I initially reported this problem [at ckeditor](https://github.com/ckeditor/ckeditor-releases/issues/74), and they referred me here.

**Comment by wsc-julia-shaptala** [comment]
Hi @berenddeboer ,

Thank you for the issue reported. We do confirm that we are able to reproduce the described problem. We plan to include the fix to one of our upcoming releases - 4.9.5. The preliminary ETA is mid December 2017. However, you shouldn't experience the problem further as now the WebSpellChecker Cloud is stabilized.

As soon as the fix is released, we will inform you.

Sincerely,

**Comment by wsc-julia-shaptala** [comment]
Hi, this issue was fixed as a part of release 4.9.5 in December 2017.
